### PR TITLE
aaaaaaaaaaaah

### LINKED ISSUE #5: Explosions can pass through indestructable entities

### PR DESCRIPTION
**Pull request by Rasengangstarr**

### CODE REVIEW

File: cursesman/entities.py [hunk 1]
```diff
@@ -92,8 +95,8 @@ def __init__(self, name, x, y, col=0):
         super().__init__(name, x, y, col=col)
         self.speed = 1
         # maybe not all characters will have the ability to drop bombs
-        self.max_bombs = 1
```
Comment (djentleman) [inline cursesman/entities.py]: i think you start with 1

File: cursesman/entities.py [hunk 2]
```diff
@@ -214,21 +217,27 @@ def burnFuse(self):
...
+        #this way the list is ordered in directional groups, from the inside out.
+        logging.warning("power = " + str(self.power))
+        explosions = []
+        
+        #left explosions
+        for p in range (1, self.power+1):
+            explosions.append(Explosion(self.x-4*p, self.y, col=self.col))
+            explosions.append(Explosion(self.x+4*p, self.y, col=self.col))
+            explosions.append(Explosion(self.x, self.y-4*p, col=self.col))
+            explosions.append(Explosion(self.x, self.y+4*p, col=self.col))
```
Comment (djentleman) [inline cursesman/entities.py]: we dont make an explosion on the bimbino, you should add an extra line before the loop or something

File: cursesman/main.py [hunk 3]
```diff
@@ -106,6 +109,51 @@ def render_game_over(stdscr):
             break
         stdscr.refresh()
 
+def handle_exploded_bombs(room, player):
```
Comment (djentleman) [inline cursesman/main.py:112]: nice

File: cursesman/entities.py (hunk 1 above)
Comment (Rasengangstarr) [reply cursesman/entities.py]: Ahh sorry, modified for debugging purposes

**Pull request merged**